### PR TITLE
feat: merge wasm into main

### PR DESCRIPTION
**Pull request by shiki-tak**
Merge wasm feature into main branch

## Checklist:

- [ ] I followed the [contributing guidelines](https://github.com/line/lfb/blob/master/CONTRIBUTING.md).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by brew0722** [comment]
What reason to merge the wasm branch first before the 0.14.0 migration is completed?

**Comment by shiki-tak** [comment]
> What reason to merge the wasm branch first before the 0.14.0 migration is completed?

I think it makes any difference either way if you merge to the main branch and then update to 0.14.0.
Do you have any concerns?

**Comment by brew0722** [comment]
no. No worries at all.
Just I was wonder what reason because just merge into main of a sudden.

As you said, there is no problem even if the migration proceeds after wasm branch merging.

**Comment by shiki-tak** [comment]
> no. No worries at all.
> Just I was wonder what reason because just merge into main of a sudden.
> 
> As you said, there is no problem even if the migration proceeds after wasm branch merging.

I see. Merge into the main branch so that we can measure the performance of vm.

File: Dockerfile
Comment (whylee259) [inline Dockerfile:54]:
```suggestion
RUN BUILD_TAGS=muslc make install CGO_ENABLED=1 LFB_BUILD_OPTIONS="$LFB_BUILD_OPTIONS"
```

**Review by brew0722 — APPROVED** [review]
LGTM, It would be nice if wasm cli test was also added to `cli_test.go` in the next PR.

**Review by whylee259 — APPROVED** [review]

**Pull request merged**
